src/adversarial/neurify.jl

```diff
@@ -224,6 +272,7 @@ function upper_bound(map::Vector{Float64}, input::HPolytope)
     model =Model(with_optimizer(GLPK.Optimizer))
     @variable(model, x[1:m])
     @constraint(model, [i in 1:n], input.constraints[i].a' * x <= input.constraints[i].b)
+    x = model[:x]
```

Comment:
Is this just for clarity, since the binding for x should already exist after `@variable`?

Reply:
It's just a redundancy, I will remove it.